File: Assets/VirtualCity/ProtoDefine/RspRemoveHousePartMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]



public class RspRemoveHousePartMessage {
[ProtoMember(1)]
    public List<HouseParts> parts;
[ProtoMember(2)]
    public int code;
[ProtoMember(3)]
    public string tip;



    public List<HouseParts> getParts() {
        return parts;
    }

    public void setParts(List<HouseParts> parts) {
        this.parts = parts;
    }


    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public string getTip() {
        return tip;
    }

    public void setTip(string tip) {
        this.tip = tip;
    }

    public RspRemoveHousePartMessage() {
    }

    public RspRemoveHousePartMessage(int code, string tip) {
        this.code = code;
        this.tip = tip;
    }

    public RspRemoveHousePartMessage(List<HouseParts> parts, int code, string tip) {
        this.parts = parts;

        this.code = code;
        this.tip = tip;
    }
}
}

File: Assets/VirtualCity/ProtoDefine/Java/building/RspRemoveHousePartMessage.java
package com.kingston.jforgame.server.game.building.message;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.building.BuildingDataPool;
import com.kingston.jforgame.server.game.building.entity.House;
import com.kingston.jforgame.server.game.building.entity.HouseParts;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

import java.util.List;

@MessageMeta(module = Modules.BULIDING,cmd = BuildingDataPool.RSP_REMOVEHOUSEPART)
public class RspRemoveHousePartMessage extends Message {
    @Protobuf(order = 1,fieldType = FieldType.OBJECT)
    private List<HouseParts> parts;
    @Protobuf(order = 2)
    private int code;
    @Protobuf(order = 3)
    private String tip;



    public List<HouseParts> getParts() {
        return parts;
    }

    public void setParts(List<HouseParts> parts) {
        this.parts = parts;
    }


    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getTip() {
        return tip;
    }

    public void setTip(String tip) {
        this.tip = tip;
    }

    public RspRemoveHousePartMessage() {
    }

    public RspRemoveHousePartMessage(int code, String tip) {
        this.code = code;
        this.tip = tip;
    }

    public RspRemoveHousePartMessage(List<HouseParts> parts, int code, String tip) {
        this.parts = parts;

        this.code = code;
        this.tip = tip;
    }
}
